__tests__/auth/auth.test.js

```diff
@@ -63,5 +63,50 @@ describe('API - Auth', () => {
 				}));
 	});
 
-	describe('Auth - Login', () => {});
+	describe('Auth - Login', () => {
+		test('Login route should expect email and password', done =>
+			request.post('/api/auth/login').end((err, res) => {
+				if (err) throw err;
+				expect(res.body.email).toBe('Email field is required');
+				expect(res.body.password).toBe('Password field is required');
+				done();
+			}));
```

Comment:
Not critical, but empty line after each test would be nice 😄

Reply:
done